contracts/mixnet/src/contract.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright 2021 - Nym Technologies SA <contact@nymtech.net>
 // SPDX-License-Identifier: Apache-2.0
 
+use std::u128;
```

Comment:
I was under impression that unless we compile with `no_std`, all of `std` things are available to us without explicit imports